src/js/locale/i18n.tsx

```diff
@@ -0,0 +1,17 @@
+import i18n from 'i18next';
+import { initReactI18next } from 'react-i18next';
+import en from './en.json';
+
+const translation = i18n.createInstance();
+
+translation.use(initReactI18next).init({
+  debug: true,
+  fallbackLng: 'en',
+  interpolation: {
+    escapeValue: false, // not needed for react as it escapes by default
+  },
+  lng: 'en',
```

Comment:
Do we want to:
`lng: window.localStorage.getItem('lng') || 'en',`

Reply:
Yes. But I don't know if it's the ideal way to tell ray what language to use

Reply:
Yeah. Also, I don't know what the decision was on whether we'd use the browser to detect the language or if we would set it based on the stored user value. We'd probably need to align this with the platform to make sure we're getting the same value.